clients/client-shell/cmds/version/version.go

```diff
@@ -18,13 +47,69 @@ var (
 
 	// VersionNumber is a formatted string with the version information. This is
 	// filled in by `yarn release`
-	VersionNumber = "24.1.10"
+	VersionNumber = "24.1.9"
 )
 
 func init() {
 	root.Command.AddCommand(Command)
+	root.Command.AddCommand(Updcommand)
 }
 
+
 func printVersion(cmd *cobra.Command, _ []string) {
 	fmt.Fprintf(cmd.OutOrStdout(), "taskcluster version %s\n", VersionNumber)
 }
+
+func check(cmd *cobra.Command, _ []string) {
+
+    //Get response and also check for error in  from git's REST API
+
+    response, err := http.Get ("https://api.github.com/repos/taskcluster/taskcluster/releases/latest") 
+
+    if err != nil {
+        fmt.Print(err.Error())
+        
+    }
+
+    // Read the whole response body and check for any errors
+
+
+    s, err := ioutil.ReadAll(response.Body)
+    
+    if err != nil {
+        fmt.Errorf(err.Error())
+        
+    }
+
+    //Create an object for the struct to parse the json data into given structure
+    
+    R := Release{}
+    if err := json.Unmarshal([]byte(s), &R); err != nil {
+        fmt.Errorf(err.Error())
+    }
+
+    //Check if taskcluster is already up to date
+
+	if R.Name == VersionNumber {
```

Comment:
When running a test just now it seems that `VersionNumber` is 24.1.9 (for me) and `R.Name` is "v24.1.10", so with the "v" prefix I'm not sure this check will ever be true.

Reply:
Done